### PR TITLE
feat(cli): add test for defaultConfigFile

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR adds a test for `defaultConfigFile` in `src/node/cli.ts`.

image

Fixes N/A

TODOS 
- [x] export function
- [x] add JSDoc block to function
- [x] write basic test

### CODE REVIEW

File: src/node/cli.ts [hunk 1]
```diff
@@ -502,10 +502,21 @@ export async function setDefaults(cliArgs: Args, configArgs?: ConfigArgs): Promi
...
+ * - bind-addr: 127.0.0.1:8080
+ * - auth: password
+ * - password: <password>
+ * - cert: false
+ */
+export function defaultConfigFile(password: string): string {
   return `bind-addr: 127.0.0.1:8080
 auth: password
-password: ${await generatePassword()}
+password: ${password}
```
Comment (jsjoeio) [inline src/node/cli.ts:519]: 🛠️ I believe functions should be _pure_ (unless they can't for some reason) so I refactored this to make `password` a param and pass in `await generatePassword()` when we call this function.

Comment (bryphe-coder) [reply src/node/cli.ts:519]: Nice!

File: test/unit/node/cli.test.ts [hunk 2]
```diff
@@ -642,3 +643,15 @@ describe("bindAddrFromArgs", () => {
     resetValue()
   })
 })
+
+describe("defaultConfigFile", () => {
+  it("should return the dfeault config file as a string", async () => {
```
Comment (bryphe-coder) [inline test/unit/node/cli.test.ts:648]:
```suggestion
  it("should return the default config file as a string", async () => {
```

**Review by bryphe-coder — APPROVED** [review]
LGTM!

File: test/unit/node/cli.test.ts (hunk 2 above)
Comment (jsjoeio) [reply test/unit/node/cli.test.ts:648]: lol nice catch, thank you!

**Review by vapurrmaid — COMMENTED** [review]
Looks great ✔️

**Pull request merged**